ndc-sqlgen/src/main/kotlin/io/hasura/ndc/sqlgen/BaseGenerator.kt

```diff
@@ -241,4 +215,27 @@ sealed interface BaseGenerator {
         }
     }
 
+    fun handleScalarComp(column: Field<Any>, value: ComparisonValue.ScalarComp): Condition {
+        return when (val scalarValue = value.value) {
+            is List<*> -> {
+                if (scalarValue.isEmpty()) {
+                    column.`in`(
```

Comment:
I see where you got this, but I think this can be simplified to 

```
 if (scalarValue.isEmpty()) {
                    DSL.falseCondition()
```
no?